sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
We should distinguish between the terms CCA and CSMA. CCA only senses the channel for an ongoing transmission.  CSMA/CA, however, is the complete process that includes backoff handling. CCA is rather a part of the CSMA/CA process.

The differences between this device and the kw2x device is that the kw2x device does not integrate the mechanisms for backoff handling in hardware. Though it integrates an automatic CCA check before sending out a packet. Thats the reason why I had chosen the name AUTOCCA.

What do you think of renaming NETCONF_OPT_AUTOCCA to NETCONF_OPT_CCA and introduce NETCONF_OPT_CSMA as a new option. The kw2x device e.g. does not implement the option NETCONF_OPT_CSMA. This would allow to en/disable SW-CSMA in the MAC-layer according to the devices' capabilities.